client/blocks/login/index.jsx

```diff
@@ -18,6 +20,11 @@ class Login extends Component {
 		twoFactorEnabled: PropTypes.bool,
 	};
 
+	state = {
+		usernamePasswordValid: false,
+		rememberMe: false,
```

Comment:
Pretty minor, but what do you think about making these property names a little clearer? Something like `hasSubmittedValidCredentials` and `isRememberMeChecked`?